pystac/item.py

```diff
@@ -330,7 +330,7 @@ def __init__(self, href, title=None, media_type=None, properties=None):
         self.properties = properties
 
         # The Item which owns this Asset.
-        self.item = None
+        self.owner = None
```

Comment:
Can you also update the Attributes PyDocs above?

Reply:
09f883a